api/valor_api/backend/metrics/detection.py

```diff
@@ -312,9 +415,9 @@ def _annotation_type_to_column(
             pd.c.annotation_id.label("pd_ann_id"),
             pd.c.score.label("score"),
         )
-        .select_from(gt)  # type: ignore - SQLAlchemy type issue
-        .join(
-            pd,  # type: ignore - SQLAlchemy type issue
+        .select_from(pd)  # type: ignore - SQLAlchemy type issue
+        .outerjoin(
```

Comment:
we make this an `outerjoin` so we can identify false predictions that aren't attached to any ground truths via the `false_positive_entries` object. this operation doesn't affect the existing AP/AR metrics since we filter out these cases from those calculations in line 465.